Library/Homebrew/test/bottle_hooks_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "formula_installer"
+require "hooks/bottles"
+
+RSpec::Matchers.alias_matcher :pour_bottle, :be_pour_bottle
+
+describe Homebrew::Hooks::Bottles do
+  subject { formula_installer }
+
+  let(:formula_installer) { FormulaInstaller.new formula }
+
+  let(:formula) {
+    double(
+      bottle: nil,
+      local_bottle_path: nil,
+      bottle_disabled?: false,
+      some_random_method: true
```

Comment:
Perhaps that's something we should re-visit? Trailing commas are very handy when argument lists are on multiple lines IMO.